packages/cli/src/services/config-service.ts

```diff
@@ -1,18 +1,27 @@
 import { BoosterApp, BoosterConfig } from '@boostercloud/framework-types'
 import * as path from 'path'
-import { exec } from 'child-process-promise'
-import { wrapExecError } from '../common/errors'
+import { guardError } from '../common/errors'
 import { checkItIsABoosterProject } from './project-checker'
 import { currentEnvironment } from './environment'
 import { createSandboxProject, removeSandboxProject } from '../common/sandbox'
-import { installProductionDependencies } from './dependencies'
+import { PackageManagerService } from './package-manager'
+import { gen, unsafeRunEffect } from '@boostercloud/framework-types/src/effect'
+import { LivePackageManager } from './package-manager/live.impl'
 
-export const DEPLOYMENT_SANDBOX = '.deploy'
+export const DEPLOYMENT_SANDBOX = path.join(process.cwd(), '.deploy')
 
 export async function createDeploymentSandbox(): Promise<string> {
   const config = await compileProjectAndLoadConfig(process.cwd())
   const sandboxRelativePath = createSandboxProject(DEPLOYMENT_SANDBOX, config.assets)
-  await installProductionDependencies(sandboxRelativePath)
+  const effect = gen(function* ($) {
+    const { setProjectRoot, installProductionDependencies } = yield* $(PackageManagerService)
+    yield* $(setProjectRoot(sandboxRelativePath))
+    yield* $(installProductionDependencies())
+  })
+  await unsafeRunEffect(effect, {
```

Comment:
The idea is that in the future we're not using `unsafeRunEffect` all over the place, but rather have a single call at the top level.